testsuite/domain/src/test/java/org/jboss/as/test/integration/domain/AbstractSSLMasterSlaveTestCase.java

```diff
@@ -117,6 +122,23 @@ protected static void checkHostStatusOnMaster(String host) throws Exception {
         Assert.fail("Cannot validate host '" + host + "' is running");
     }
 
+    protected static void checkHostStatusOnMasterOverRemote(String host) throws IOException, InterruptedException {
+        try (ModelControllerClient mcc = ModelControllerClient.Factory.create(
+                InetAddress.getByName(TestSuiteEnvironment.getServerAddress()), 9999, Authentication.getCallbackHandler())) {
```

Comment:
Why create a client for this? DomainTestSupport.getDomainMasterLifecycleUtil().getDomainClient() should provide a client.